### PR TITLE
Added 404 with links

### PR DESCRIPTION
**Pull request by stevenGravy**
adds a 404 page to have this instead of a generic 404 error.

image

### CODE REVIEW

File: pages/404.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// 404.js
+
+export default function FourOhFour() {
+  return (
+    <>
+      <h1>404 - Page Not Found</h1>
+      We&apos;re very sorry - we could not find the page you were looking for.
+      Please navigate to the Teleport{" "}
+      <a href="https://docs.goteleport.com">Documentation</a> or the{" "}
```
Comment (reedloden) [inline pages/404.js:9]:
```suggestion
      <a href="/docs/">Documentation</a> or the{" "}
```

File: pages/404.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      <a href="https://goteleport.com">Home Page</a> to find what you&apos;re
```
Comment (reedloden) [inline pages/404.js:10]:
```suggestion
      <a href="/">Home Page</a> to find what you&apos;re
```

**Review by reedloden — COMMENTED** [review]
Prefer relative URLs so that things work on staging.

**Comment by reedloden** [comment]
Shouldn't we make it look similar to https://goteleport.com/pagedoesnotexist/? More branding and such?

**Comment by stevenGravy** [comment]
> Shouldn't we make it look similar to https://goteleport.com/pagedoesnotexist/? More branding and such?

We can. This was an intial attempt to get something better then the current:

image

**Review by reedloden — APPROVED** [review]

**Pull request merged**